src/rgw/rgw_lc.cc

```diff
@@ -13,6 +13,7 @@
 #include "rgw_common.h"
 #include "rgw_bucket.h"
 #include "rgw_lc.h"
+#include "common/ceph_json.h"
```

Comment:
why include this header here? the dump() method is defined in  src/rgw/rgw_json_enc.cc.

Reply:
@tchaikov  I defined dump() method in other file previously, then forgot to remove it. so... 
Thanks anyway, check pls.